jgiven-spock/build.gradle

```diff
@@ -8,10 +8,9 @@ description = "Module for writing JGiven tests with Spock"
 
 dependencies {
     api project(':jgiven-junit')
-    implementation 'org.codehaus.groovy:groovy:3.0.8'
-    implementation 'org.spockframework:spock-core:2.0-M4-groovy-3.0'
-    implementation "org.spockframework:spock-junit4:2.0-M5-groovy-3.0"
-    testCompile project(':jgiven-html5-report')
+    implementation 'org.codehaus.groovy:groovy:2.5.13'
+    implementation('org.spockframework:spock-core:1.3-groovy-2.5')
+    testImplementation project(':jgiven-html5-report')
```

Comment:
As far as I know brackets shouldn't make any difference, or? In this case we should stick to a unified notion.